test/github_push_retest_test.go

```diff
@@ -100,20 +106,28 @@ func TestGithubPushRequestGitOpsCommentCancel(t *testing.T) {
 	repo, _ := twait.UntilRepositoryUpdated(ctx, g.Cnx.Clients, waitOpts) // don't check for error, because canceled is not success and this will fail
 	cancelled := false
 	for _, c := range repo.Status {
-		if c.Conditions[0].Reason == "Cancelled" {
+		if c.Conditions[0].Reason == tektonv1.TaskRunReasonCancelled.String() {
 			cancelled = true
 		}
 	}
-	assert.Assert(t, cancelled, "No cancelled status in repo Statuses run found")
-	assert.Equal(t, repo.Status[len(repo.Status)-1].Conditions[0].Status, corev1.ConditionFalse)
+
+	// this went too fast so at least we check it was requested for it
+	if !cancelled {
+		reg := regexp.MustCompile(".*pipelinerun.*is done.*skipping cancellation.*")
+		err = twait.RegexpMatchingInControllerLog(ctx, g.Cnx, *reg, 10, "controller", github.Int64(20))
+		if err != nil {
+			t.Errorf("neither a cancelled pipelinerun in repo status or a request to skip the cancellation in the controller log was found: %s", err.Error())
+		}
+		return
```

Comment:
If neither of the things happened then we should fail the test right as its an indication that `/cancel` did not work so there might be issue with code :thinking: